src/views/opm.tsx

```diff
@@ -312,226 +315,265 @@ const OPM: React.FC = () => {
     setDisabled(formFields.map((e) => e.value).filter(Boolean).length === 0);
   }, [formFields]);
 
+  const handleOPMExpandClick = () => {
+    navigate("/opm");
+  };
+
+  const getConfigOptions = () => {
+    const chartOptions = JSON.parse(JSON.stringify(options));
+    chartOptions.layout.padding = 0;
+    chartOptions.scales.x.title.padding.top = 10;
+    return chartOptions;
+  };
+
   return (
     <>
-      <div className={`flex gap-[66vw]`}>
-        <p className="font-bold w-[3vw] mt-[3.9vh] ml-[3vw] text-[#F2F2F2]">
-          OPM
-        </p>
-        <CustomImage
-          src={FilterIcon}
-          className="w-[2.34vw] self-end"
-          alt="Filter Icon"
-          onClick={onFilterClickHandler}
-        />
-      </div>
-      {showFilters && (
+      {location.pathname.includes("home") && data && (
+        <div className="w-full sm:w-1/2 bg-[#30343B] p-4 rounded-lg">
+          <div className="flex justify-between mb-3 items-center">
+            <span className="text-[#F2F2F2] font-bold text-lg font-helvetica">
+              OPM
+            </span>
+            <div>
+              <button
+                className="rounded-full pr-2"
+                onClick={handleOPMExpandClick}
+              >
+                <CustomImage src={open_in_full_window} />
+              </button>
+            </div>
+          </div>
+          {/* <div className="bg-[#30343B] border-0 rounded-[10px] w-[71.74vw] ml-[2.85vw] h-[62.23vh] mt-[3vh]">
```

Comment:
Can you remove the commented lines if not needed ?

Reply:
removed